<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HARDIS Matériel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ordinateur et accessoires</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bureau</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chaise</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Micro-ondes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cafetière</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Titre">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Enregistrer un nouvel équipement</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Type">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Type de matériel</string>
    </property>
   </widget>
   <widget class="QLabel" name="BU">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BU</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BU_txt">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Equipe_txt">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Equipe">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Marque_txt">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Marque">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Marque</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NoSerie_txt">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="NoSerie">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Numéro de série</string>
    </property>
   </widget>
   <widget class="QLabel" name="DateAchat">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Date d'achat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Responsable_txt">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>270</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Responsable">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Personne
responsable</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="Date">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
    <property name="date">
     <date>
      <year>2018</year>
      <month>5</month>
      <day>23</day>
     </date>
    </property>
   </widget>
   <widget class="QPushButton" name="Enregistrer">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>460</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Enregistrer</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>282</width>
      <height>160</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>1</width>
      <height>2</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHardis_Mat_riel">
    <property name="title">
     <string>Hardis Matériel</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEnregistrer_mat_riel">
    <property name="title">
     <string>Enregistrer matériel</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuD_placement_mat_riel">
    <property name="title">
     <string>Déplacement matériel</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuConsulter_mat_riel">
    <property name="title">
     <string>Consulter matériel</string>
    </property>
   </widget>
   <addaction name="menuHardis_Mat_riel"/>
   <addaction name="menuEnregistrer_mat_riel"/>
   <addaction name="menuD_placement_mat_riel"/>
   <addaction name="menuConsulter_mat_riel"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
